RecoVertex/BeamSpotProducer/test/testBeamSpotCompatibility_cfg.py

```diff
@@ -0,0 +1,102 @@
+import sys
+FLOAT_MAX = 3.402823466E+38
+
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+####################################################################
+# Options
+####################################################################
+options = VarParsing.VarParsing()
+options.register('dbFromEvent',
+                 False, # default value
+                 VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                 VarParsing.VarParsing.varType.bool, # string, int, or float
+                 "use the Event for reading the DB?")
+options.register('warningThreshold',
+                 1., # default value
+                 VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                 VarParsing.VarParsing.varType.float, # string, int, or float
+                 "threshold to emit a warning")
+options.register('errorThreshold',
+                 3., # default value
+                 VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                 VarParsing.VarParsing.varType.float, # string, int, or float
+                 "threshold to emit a warning")
+options.parseArguments()
+
+import FWCore.ParameterSet.Config as cms
+process = cms.Process("testBeamSpotCompatibility")
+
+####################################################################
+# Load source
+####################################################################
+readFiles = cms.untracked.vstring()
+readFiles.extend(['/store/data/Run2023D/HLTPhysics/ALCARECO/TkAlMinBias-PromptReco-v1/000/370/580/00000/acdddb09-046c-4375-82f3-678138106ac7.root'])
```

Comment:
is this file from some special case of bad saved result?
or just something random.
My concern is also about the file availability long term.

Reply:
> is this file from some special case of bad saved result?

it's just a random file from last year that I know it's not covered by the tag in the Global Tag (the use case actually was presented by Francesco @francescobrivio, so he can comment more).

>  My concern is also about the file availability long term.

I don't think that's a problem. The file is currently at `T2_CH_CERN` and as soon as this is merged it'll get cached.

Reply:
I am going to resolve this comment. Let me know if you think otherwise.